daemon/config/builder.go

```diff
@@ -135,6 +135,7 @@ type BuilderHistoryConfig struct {
 type BuilderEntitlements struct {
 	NetworkHost      *bool `json:"network-host,omitempty"`
 	SecurityInsecure *bool `json:"security-insecure,omitempty"`
+	Device           *bool `json:"device,omitempty"`
 }
```

Comment:
This probably needs changes in the daemon docs, although it looks like we currently don't document these at all? https://github.com/docker/cli/blob/7668b683d2bdd470b6fc95082f4a1f9f3f5c756a/docs/reference/dockerd.md#on-linux

currently only shows this in the example (no docs);

```json
  "builder": {
    "gc": {
      "enabled": true,
      "defaultKeepStorage": "10GB",
      "policy": [
        { "keepStorage": "10GB", "filter": ["unused-for=2200h"] },
        { "keepStorage": "50GB", "filter": ["unused-for=3300h"] },
        { "keepStorage": "100GB", "all": true }
      ]
    }
  },
```